spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe ApplicationController do
+  before :each do
+    @current_session = mock_authenticated_session
+  end
+
+  controller do
+    def index
+      render inline: 'hello world'
+    end
+  end
+
+  describe 'user session' do
+    it 'extends life of session each time user makes a request' do
+      initial_session_time = @current_session.updated_at
+      sleep 1
```

Comment:
that was my first instinct too.

alas, the feature depends on the Redis server, which is completely independent of Timecop and of Ruby.